README.md

```diff
@@ -87,6 +87,6 @@ See [LICENSE.md](LICENSE.md)
 
 We'd be quite excited if you'd like to contribute to Worldview Timeline Components! Whether it's finding bugs, adding new features, fixing anything broken, or improving documentation, get started by submitting an issue or pull request!
```

Comment:
Need to change one more reference to "Worldview Timeline Components" here in line 88